ClientApp/src/components/admin/overview/OverviewMacro.tsx

```diff
@@ -17,12 +17,25 @@ import { Alert } from "@material-ui/lab";
 
 const initState: SelectedConnectionType = {
   giver: undefined,
-  recipient: {} as RecipientType,
+  recipient: undefined,
 };
 interface IOverviewMacro {
   location: string;
   accessToken: string;
 }
+
+const style = {
```

Comment:
Please move this into the useStyles file, imported at line 13.

Name the style to fit its usage, then change line 168 to `<Box sx={classes.myStyle}>`